src/Theme/RegionSelector.elm

```diff
@@ -2,10 +2,10 @@ module Theme.RegionSelector exposing (Msg(..), viewRegionSelector)
 
 import Copy.Keys exposing (Key(..))
 import Copy.Text exposing (t)
-import Css exposing (Style, active, auto, backgroundColor, batch, borderBox, borderColor, borderRadius, borderStyle, borderWidth, boxSizing, center, color, cursor, displayFlex, fitContent, flexWrap, focus, fontSize, fontWeight, hover, int, justifyContent, listStyleType, margin2, margin4, maxWidth, none, padding4, pointer, position, property, px, relative, rem, solid, textAlign, width, wrap)
+import Css exposing (Style, active, auto, backgroundColor, batch, borderBox, borderColor, borderRadius, borderStyle, borderWidth, boxSizing, center, color, cursor, display, displayFlex, fitContent, flexWrap, focus, fontSize, fontWeight, hover, int, justifyContent, listStyleType, margin2, margin4, maxWidth, none, padding4, pointer, position, property, px, relative, rem, solid, textAlign, width, wrap)
 import Css.Global exposing (descendants, typeSelector)
 import Css.Transitions exposing (transition)
-import Data.PlaceCal.Partners
+import Data.PlaceCal.Partners exposing (partnershipTagList)
```

Comment:
We've been fully qualifying calls instead of exposing to make it more obvious where the code comes from on the line it's used.

e.g. call the function on line 98 with `Data.PlaceCal.Partners.partnershipTagList` and remove the exposing clause.